Cargo.toml

```diff
@@ -22,15 +22,18 @@ bench = true
 doc = true
 
 [features]
-default = ["std"]
+default = ["std", "runtime-rng"]
```

Comment:
From the description it sounds like your goal is to remove dependencies when `--no-default-features` is specified. So why is a feature being removed from the default set? It seems like this would not affect this goal, and it is highly undesirable, because it breaks backwards compatibility and forces all existing users to add the feature flag when upgrading to maintain current behavior.